### ISSUE TITLE
 Expected object 'SharedRes' has no actual declaration in module <composeApp_debug> for JVM

### ISSUE THREAD

**Issue by pankaj1920** [icerockdev/moko-resources#606]
Expected object 'SharedRes' has no actual declaration in module <composeApp_debug> for JVM

**Comment by pankaj1920** [comment]
sourceSets {
        getByName("main").java.srcDirs("build/generated/moko/androidMain/src")
    }
    
    after adding this it is working but localization is not changing realtime as i change the device language from setting. i have to close app and reopen then it is working

**Comment by ScottPierce** [comment]
I'm seeing a similar problem with multiplatform. The work around was:

```Kotlin
    sourceSets {
        getByName("androidMain") {
            this.kotlin.srcDir("build/generated/moko/androidMain/src")
        }
    }
```

**Comment by ScottPierce** [comment]
> Thank for the solution this have solve my problems too :) ✅
> 
> I think we can close this ticket ❌

I don't think so. This should be fixed in the plugin. The workaround shouldn't be necessary.

**Comment by Bradleycorn** [comment]
I'm also seeing this issue. Thanks for the workaround @ScottPierce.
I agree this should have a proper fix

**Comment by avently** [comment]
[Alex009](https://github.com/Alex009)
>fix in https://github.com/icerockdev/moko-resources/pull/575

Don't think it's fixed because I have similar error message on **0.24.0-alpha-5**:
```log
Expected object 'MR' has no actual declaration in module <common> for JVM
```

In my case I have android, common, desktop source sets. On Desktop build I see this error, Android build works fine. desktopMain directory is not created at all in common/build/generated/moko. So workaround with srcSet will not work because of this.

**Comment by Alex009** [comment]
@avently please preapre reproducer, because in our samples all works

**Comment by ExNDY** [comment]
will be fixed in 0.24.0, please check in 0.24.0-alpha-6

**Comment by Alex009** [comment]
should be fixed in 0.24.0-beta-1
